jbpm-flow/src/test/java/org/jbpm/process/core/timer/BusinessCalendarImplTest.java

```diff
@@ -387,6 +389,26 @@ public void testCalculateMinutesPassingWeekend() {
         assertEquals(expectedDate, formatDate("yyyy-MM-dd HH:mm:ss", result));
     }
 
+    @Test
+    public void testLoadGoodInputStream() throws Exception {
+        URL url = this.getClass().getResource("/business-custom-calendar.properties");
+        File file = new File(url.toURI());
+
+        BusinessCalendarImpl businessCal = new BusinessCalendarImpl(file.getAbsolutePath());
+        int myCustomProperty = businessCal.getPropertyAsInt("MyCustomProperty", "3");
+
+        assertEquals(7, myCustomProperty);
+    }
+
+    @Test
+    public void testLoadBadInputStream() throws Exception {
```

Comment:
there is no checked exception thrown by this method, please remove.
```suggestion
    public void testLoadBadInputStream() {
```

Reply:
done